[Bug] iOS ImageRenderer NullReferenceException

<!-- Bug report best practices: https://github.com/xamarin/Xamarin.Forms/wiki/Submitting-Issues -->

### Description
iOS ImageRenderer NullReferenceException

### Steps to Reproduce

1.
XAML: 
```
<Image Source="{Binding Base64Image, Converter={StaticResource base64ImageConverter}}" 
                    WidthRequest="66" HeightRequest="44"
                    HorizontalOptions="Center" VerticalOptions="Center"/>

```

2. 
C#:
```
public class Base64ImageConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
        {
            if (!(value is string base64)) return null;

            var bytes = System.Convert.FromBase64String(base64);
            return ImageSource.FromStream(() => new MemoryStream(bytes));
        }

        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
        {
            throw new NotSupportedException("Only one way bindings are supported with this converter");
        }
    }

```

3. 
Run on emulator: IPhone 13 iOS 15.2


### Expected Behavior
Doesn't throw Exceptions

### Actual Behavior
Object reference not set to an instance of an object

  at Xamarin.Forms.Platform.iOS.ImageRenderer.UpdateBackground () [0x00000] in D:\a\1\s\Xamarin.Forms.Platform.iOS\Renderers\ImageRenderer.cs:123 
  at Xamarin.Forms.Platform.iOS.ImageRenderer.OnElementChanged (Xamarin.Forms.Platform.iOS.ElementChangedEventArgs`1[TElement] e) [0x000a8] in D:\a\1\s\Xamarin.Forms.Platform.iOS\Renderers\ImageRenderer.cs:74 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) [0x00000] in /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/src/Xamarin.iOS/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/AsyncMethodBuilder.cs:1021 
  at Foundation.NSAsyncSynchronizationContextDispatcher.Apply () [0x00000] in /Users/builder/azdo/_work/1/s/xamarin-macios/src/Foundation/NSAction.cs:178 
--- End of stack trace from previous location where exception was thrown ---

  at (wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain(int,string[],intptr,intptr)
  at UIKit.UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName) [0x00013] in /Users/builder/azdo/_work/1/s/xamarin-macios/src/UIKit/UIApplication.cs:73 
  at Multipass.iOS.Application.Main (System.String[] args) [0x00008] in C:\Source\Repos\ClientMobileApp\Multipass.iOS\Main.cs:21 



### Basic Information

- Version with issue: 5.0.0.2337
- Last known good version: 5.0.0.2291
- Platform Target Frameworks: <!-- All that apply -->
  - crossplatform project:
     Target Framework .NET Standard 2.0
  - iOS:
    MinimumOSVersion 10.0
    MtouchSdkVersion 10.3

### Environment

<!--
1.
Visual Studio: Help > About Microsoft Visual Studio > Copy Info [button]
Visual Studio for Mac: Visual Studio > About Visual Studio > Show Details > Copy Information [button]
2. Paste into the code block below (between ```)
-->

<details><summary>Show/Hide Visual Studio info</summary>

```
Microsoft Visual Studio Community 2022
Version 17.0.5
VisualStudio.17.Release/17.0.5+32112.339
Microsoft .NET Framework
Version 4.8.04084

Installed Version: Community

ASP.NET and Web Tools 2019   17.0.795.42246
ASP.NET and Web Tools 2019

Azure App Service Tools v3.0.0   17.0.795.42246
Azure App Service Tools v3.0.0

C# Tools   4.0.1-1.21568.1+6ab6601178d9fba8c680b56934cd1742e0816bff
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Extensibility Message Bus   1.2.6 (master@34d6af2)
Provides common messaging-based MEF services for loosely coupled Visual Studio extension components communication and integration.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Mono Debugging for Visual Studio   17.0.11 (54f19d2)
Support for debugging Mono processes with Visual Studio.

NuGet Package Manager   6.0.1
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

TypeScript Tools   17.0.1001.2002
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   4.0.1-1.21568.1+6ab6601178d9fba8c680b56934cd1742e0816bff
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools   17.0.0-beta.21522.2+6d626ff0752a77d339f609b4d361787dc9ca93a5
Microsoft Visual F# Tools

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

Visual Studio IntelliCode   2.2
AI-assisted development for Visual Studio.

VisualStudio.DeviceLog   1.0
Information about my package

VisualStudio.Foo   1.0
Information about my package

VisualStudio.Mac   1.0
Mac Extension for Visual Studio

Xamarin   17.0.0.343 (d17-0@fb07a17)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   17.0.0.183 (remotes/origin/d17-0@a351f0f1f)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   17.0.17 (9e779b0)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   12.1.0.5 (d17-0/6b0e6b2)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: c633fe9
    Java.Interop: xamarin/java.interop/d17-0@febb1367
    ProGuard: Guardsquare/proguard/v7.0.1@912d149
    SQLite: xamarin/sqlite/3.36.0@a575761
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d17-0@a5194e9


Xamarin.iOS and Xamarin.Mac SDK   15.4.0.0 (8fc41ae82)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.
```

</details>

### Build Logs

<!--
If your issue is related to a build error, be sure to attach build logs.

Binary logs are preferred instead of text logs.  Please save and attach a .binlog if possible:
(https://docs.microsoft.com/en-us/visualstudio/msbuild/obtaining-build-logs-with-msbuild?view=vs-2019#save-a-binary-log)

Otherwise, a text log is still helpful:

1. Place cursor below this comment block.
2. Attach build log or link to gist (https://gist.github.com/) of the log.
3. Logs can be found in the Build Output tab of the Errors pad in Visual Studio
To get full build logs, set the log verbosity to diagnostic at the following locations:
- On Visual Studio for Mac: Preferences > Projects > Build
- On Visual Studio for Windows: Tools > Options > Projects and Solutions > Build and Run
On Visual Studio for Windows you also want to add `-v -v -v -v` to the mtouch additional arguments by right-clicking the project in the solution explorer and selecting `Properties`.
-->

<details><summary>Show/Hide Build Logs</summary>

Build started...
Restored C:\Source\Repos\ClientMobileApp\Multipass.iOS.Tuke\Multipass.iOS.Tuke.csproj (in 122 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.iOS.KontronV\Multipass.iOS.KontronV.csproj (in 42 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.Tests\Multipass.Tests.csproj (in 347 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.Android.SmartPaySPB\Multipass.Android.SmartPaySPB.csproj (in 130 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.Android\Multipass.Android.csproj (in 222 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.Android.GorodPay\Multipass.Android.GorodPay.csproj (in 30 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.Android.KontronE\Multipass.Android.KontronE.csproj (in 30 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.Android.KontronV\Multipass.Android.KontronV.csproj (in 27 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.Android.MPCloud\Multipass.Android.MPCloud.csproj (in 27 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass\Multipass.csproj (in 69 ms).
1>------ Build started: Project: Multipass.ApiClient, Configuration: Debug Any CPU ------
Restored C:\Source\Repos\ClientMobileApp\Multipass.Android.Tuke\Multipass.Android.Tuke.csproj (in 29 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.iOS\Multipass.iOS.csproj (in 41 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.iOS.KontronE\Multipass.iOS.KontronE.csproj (in 44 ms).
Restored C:\Source\Repos\ClientMobileApp\Multipass.iOS.MPCloud\Multipass.iOS.MPCloud.csproj (in 42 ms).
1>Multipass.ApiClient -> C:\Source\Repos\ClientMobileApp\ApiClient\Multipass.ApiClient\bin\Debug\netstandard2.0\Multipass.ApiClient.dll
2>------ Build started: Project: Multipass, Configuration: Debug Any CPU ------
2>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2304,5): warning MSB3270: There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "Cards", "x86". This mismatch may cause runtime failures. Please consider changing the targeted processor architecture of your project through the Configuration Manager so as to align the processor architectures between your project and references, or take a dependency on references with a processor architecture that matches the targeted processor architecture of your project.
2>C:\Source\Repos\ClientMobileApp\Multipass\Helpers\PermissionHelper.cs(18,21,18,60): warning CS4014: Because this call is not awaited, execution of the current method continues before the call is completed. Consider applying the 'await' operator to the result of the call.
2>C:\Source\Repos\ClientMobileApp\Multipass\Models\HttpLoggingHandler.cs(92,33,92,35): warning CS0168: The variable 'ex' is declared but never used
2>C:\Source\Repos\ClientMobileApp\Multipass\Models\HttpLoggingHandler.cs(135,30,135,32): warning CS0168: The variable 'ex' is declared but never used
2>C:\Source\Repos\ClientMobileApp\Multipass\ViewModels\Popup\NFCReadingPopupViewModel.cs(51,43,51,45): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
2>C:\Source\Repos\ClientMobileApp\Multipass\ViewModels\Popup\NFCReadingPopupViewModel.cs(56,41,56,43): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
2>C:\Source\Repos\ClientMobileApp\Multipass\Services\NavigationService.cs(270,63,270,65): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
2>C:\Source\Repos\ClientMobileApp\Multipass\ViewModels\Popup\ChangeLanguagePopupViewModel.cs(68,25,68,27): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
2>MSBUILD : warning FodyPackageReference: Fody: The package reference for ConfigureAwait.Fody does not contain PrivateAssets='All'
2>MSBUILD : warning FodyPackageReference: Fody: The package reference for PropertyChanged.Fody does not contain PrivateAssets='All'
2>C:\Source\Repos\ClientMobileApp\Multipass\UserControls\Feedback.xaml.cs(187,9,187,10): warning : Fody/PropertyChanged: Unsupported signature for a On_PropertyName_Changed method: OnStarsValueChanged in Multipass.UserControls.Feedback. You can suppress this warning with [SuppressPropertyChangedWarnings].
2>C:\Source\Repos\ClientMobileApp\Multipass\UserControls\Feedback.xaml.cs(194,9,194,10): warning : Fody/PropertyChanged: Unsupported signature for a On_PropertyName_Changed method: OnTextChanged in Multipass.UserControls.Feedback. You can suppress this warning with [SuppressPropertyChangedWarnings].
2>C:\Source\Repos\ClientMobileApp\Multipass\UserControls\NotificationSettingsStackLayout.cs(16707566-16707566): warning : Fody/PropertyChanged: The type Multipass.UserControls.NotificationSettingsStackLayout has a On_PropertyName_Changed method (OnItemsSourceChanged) which is static. You can suppress this warning with [SuppressPropertyChangedWarnings].
2>C:\Source\Repos\ClientMobileApp\Multipass\UserControls\TariffPassStackLayout.cs(16707566-16707566): warning : Fody/PropertyChanged: The type Multipass.UserControls.TariffPassStackLayout has a On_PropertyName_Changed method (OnItemsSourceChanged) which is static. You can suppress this warning with [SuppressPropertyChangedWarnings].
2>C:\Source\Repos\ClientMobileApp\Multipass\UserControls\TariffTicketStackLayout.cs(16707566-16707566): warning : Fody/PropertyChanged: The type Multipass.UserControls.TariffTicketStackLayout has a On_PropertyName_Changed method (OnItemsSourceChanged) which is static. You can suppress this warning with [SuppressPropertyChangedWarnings].
2>C:\Source\Repos\ClientMobileApp\Multipass\Behaviors\BehaviorBase.cs(29,9,29,10): warning : Fody/PropertyChanged: Unsupported signature for a On_PropertyName_Changed method: OnBindingContextChanged in Multipass.Behaviors.BehaviorBase`1. You can suppress this warning with [SuppressPropertyChangedWarnings].
2>C:\Source\Repos\ClientMobileApp\Multipass\Behaviors\MaskedBehavior.cs(62,9,62,10): warning : Fody/PropertyChanged: Unsupported signature for a On_PropertyName_Changed method: OnEntryTextChanged in Multipass.Behaviors.MaskedBehavior. You can suppress this warning with [SuppressPropertyChangedWarnings].
2>Multipass -> C:\Source\Repos\ClientMobileApp\Multipass\bin\Debug\netstandard2.0\Multipass.dll
2>Done building project "Multipass.csproj".
3>------ Build started: Project: Multipass.iOS.KontronE, Configuration: Debug iPhoneSimulator ------
3>  Executing SayHello Task to establish a connection to a Remote Server. 
3>  			Properties: 
3>  				SessionId=4f9fe3b4a459638c119440281481a8745fe7765b69b7afd2db772c1e546c45fe, 
3>  				Addresss=192.168.50.130, 
3>  				SshPort=22, 
3>  				TcpPort=52123, 
3>  				User=admin, 
3>  				AppName=Multipass.iOS.KontronE,
3>  				VisualStudioProcessId=15128,
3>  				ContinueOnDisconnected=False
3>  Executing SayHello Task to establish a connection to a Remote Server. 
3>  			Properties: 
3>  				SessionId=4f9fe3b4a459638c119440281481a8745fe7765b69b7afd2db772c1e546c45fe, 
3>  				Addresss=192.168.50.130, 
3>  				SshPort=22, 
3>  				TcpPort=52123, 
3>  				User=admin, 
3>  				AppName=Multipass.iOS.KontronE,
3>  				VisualStudioProcessId=15128,
3>  				ContinueOnDisconnected=False
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1596,3): warning : The iOS SDK version '10.3' is not installed. Using newer version '15.2' instead'.
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1596,3): warning :         
3>  Detected signing identity:
3>          
3>    Bundle Id: com.multipass.KontronE
3>    App Id: com.multipass.KontronE
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/splash_icon.png obj/iPhoneSimulator/Debug/optimized/splash_icon.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/splash_icon@2x.png" "obj/iPhoneSimulator/Debug/optimized/splash_icon@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/splash_icon@3x.png" "obj/iPhoneSimulator/Debug/optimized/splash_icon@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/corneredbarcode2.png obj/iPhoneSimulator/Debug/optimized/corneredbarcode2.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/corneredbarcode2@2x.png" "obj/iPhoneSimulator/Debug/optimized/corneredbarcode2@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/corneredbarcode2@3x.png" "obj/iPhoneSimulator/Debug/optimized/corneredbarcode2@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/logo_bus.png obj/iPhoneSimulator/Debug/optimized/logo_bus.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/logo_bus@2x.png" "obj/iPhoneSimulator/Debug/optimized/logo_bus@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/logo_bus@3x.png" "obj/iPhoneSimulator/Debug/optimized/logo_bus@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/logo_car.png obj/iPhoneSimulator/Debug/optimized/logo_car.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/logo_car@2x.png" "obj/iPhoneSimulator/Debug/optimized/logo_car@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/logo_car@3x.png" "obj/iPhoneSimulator/Debug/optimized/logo_car@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/logo_metro.png obj/iPhoneSimulator/Debug/optimized/logo_metro.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/logo_metro@2x.png" "obj/iPhoneSimulator/Debug/optimized/logo_metro@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/logo_metro@3x.png" "obj/iPhoneSimulator/Debug/optimized/logo_metro@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/logo_tram.png obj/iPhoneSimulator/Debug/optimized/logo_tram.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/logo_tram@2x.png" "obj/iPhoneSimulator/Debug/optimized/logo_tram@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/logo_tram@3x.png" "obj/iPhoneSimulator/Debug/optimized/logo_tram@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/nfc.png obj/iPhoneSimulator/Debug/optimized/nfc.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/nfc@2x.png" "obj/iPhoneSimulator/Debug/optimized/nfc@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/nfc@3x.png" "obj/iPhoneSimulator/Debug/optimized/nfc@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/payment_options.png obj/iPhoneSimulator/Debug/optimized/payment_options.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/payment_options@2x.png" "obj/iPhoneSimulator/Debug/optimized/payment_options@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/payment_options@3x.png" "obj/iPhoneSimulator/Debug/optimized/payment_options@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/tutorial_1_barcode.png obj/iPhoneSimulator/Debug/optimized/tutorial_1_barcode.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_1_barcode@2x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_1_barcode@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_1_barcode@3x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_1_barcode@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/tutorial_2_card.png obj/iPhoneSimulator/Debug/optimized/tutorial_2_card.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_2_card@2x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_2_card@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_2_card@3x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_2_card@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/tutorial_3_phone.png obj/iPhoneSimulator/Debug/optimized/tutorial_3_phone.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_3_phone@2x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_3_phone@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_3_phone@3x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_3_phone@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/tutorial_4_ticket.png obj/iPhoneSimulator/Debug/optimized/tutorial_4_ticket.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_4_ticket@2x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_4_ticket@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_4_ticket@3x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_4_ticket@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/tutorial_5_agree.png obj/iPhoneSimulator/Debug/optimized/tutorial_5_agree.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_5_agree@2x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_5_agree@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_5_agree@3x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_5_agree@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/tutorial_6_sms.png obj/iPhoneSimulator/Debug/optimized/tutorial_6_sms.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_6_sms@2x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_6_sms@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_6_sms@3x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_6_sms@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/tutorial_7_cardpay.png obj/iPhoneSimulator/Debug/optimized/tutorial_7_cardpay.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_7_cardpay@2x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_7_cardpay@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_7_cardpay@3x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_7_cardpay@3x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 Resources/tutorial_8_keys.png obj/iPhoneSimulator/Debug/optimized/tutorial_8_keys.png 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_8_keys@2x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_8_keys@2x.png" 
3>  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/pngcrush -q -iphone -f 0 "Resources/tutorial_8_keys@3x.png" "obj/iPhoneSimulator/Debug/optimized/tutorial_8_keys@3x.png" 
3>  /usr/bin/plutil -convert binary1 -o obj/iPhoneSimulator/Debug/optimized/en.lproj/InfoPlist.strings Resources/en.lproj/InfoPlist.strings 
3>  /usr/bin/plutil -convert binary1 -o obj/iPhoneSimulator/Debug/optimized/ru.lproj/InfoPlist.strings Resources/ru.lproj/InfoPlist.strings 
3>  Consider app.config remapping of assembly "Newtonsoft.Json, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" from Version "12.0.0.0" [] to Version "13.0.0.0" [C:\Users\minaev\.nuget\packages\newtonsoft.json\13.0.1\lib\netstandard2.0\Newtonsoft.Json.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "SkiaSharp, Culture=neutral, PublicKeyToken=0738eb9f132ed756" from Version "1.68.0.0" [] to Version "2.80.0.0" [C:\Users\minaev\.nuget\packages\skiasharp\2.80.3\lib\xamarinios1.0\SkiaSharp.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Runtime.CompilerServices.Unsafe, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.4.1" [] to Version "6.0.0.0" [C:\Users\minaev\.nuget\packages\system.runtime.compilerservices.unsafe\6.0.0\lib\netstandard2.0\System.Runtime.CompilerServices.Unsafe.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "netstandard, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" from Version "2.0.0.0" [] to Version "2.1.0.0" [C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\netstandard.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Runtime, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.20.0" [] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Threading, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Threading.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Collections, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Collections.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Threading.Tasks, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Threading.Tasks.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Diagnostics.Debug, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Diagnostics.Debug.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Runtime.Extensions, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Text.Encoding, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\ReferenceAssemblies\Microsoft\Framework\Xamarin.iOS\v1.0\Facades\System.Text.Encoding.dll] to solve conflict and get rid of warning.
3>  Consider app.config remapping of assembly "System.Text.RegularExpressions, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [] to solve conflict and get rid of warning.
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2304,5): warning MSB3276: Found conflicts between different versions of the same dependent assembly. Please set the "AutoGenerateBindingRedirects" property to true in the project file. For more information, see http://go.microsoft.com/fwlink/?LinkId=294190.
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning : [xma]: An error occurred while using the SCP Client. Details: Message type 52 is not valid in the current context.
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning : Message type 52 is not valid in the current context.
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Renci.SshNet.Session.WaitOnHandle(WaitHandle waitHandle, TimeSpan timeout)
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Renci.SshNet.Channels.ChannelSession.Open()
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Renci.SshNet.ScpClient.Upload(Stream source, String path)
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Xamarin.Messaging.Ssh.MessagingScpClient.<ExecuteAsync>d__23.MoveNext() in D:\a\_work\1\s\src\Xamarin.Messaging.Ssh\MessagingScpClient.cs:line 155
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning : [xma]: An error occurred while uploading files
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning : Message type 52 is not valid in the current context.
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Renci.SshNet.Session.WaitOnHandle(WaitHandle waitHandle, TimeSpan timeout)
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Renci.SshNet.Channels.ChannelSession.Open()
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Renci.SshNet.ScpClient.Upload(Stream source, String path)
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Xamarin.Messaging.Ssh.MessagingScpClient.<ExecuteAsync>d__23.MoveNext() in D:\a\_work\1\s\src\Xamarin.Messaging.Ssh\MessagingScpClient.cs:line 155
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning : --- End of stack trace from previous location where exception was thrown ---
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Xamarin.Messaging.Ssh.MessagingScpClient.<ExecuteAsync>d__23.MoveNext() in D:\a\_work\1\s\src\Xamarin.Messaging.Ssh\MessagingScpClient.cs:line 180
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning : --- End of stack trace from previous location where exception was thrown ---
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Xamarin.Messaging.Ssh.MessagingScpClient.<ExecuteAsync>d__23.MoveNext() in D:\a\_work\1\s\src\Xamarin.Messaging.Ssh\MessagingScpClient.cs:line 175
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning : --- End of stack trace from previous location where exception was thrown ---
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Xamarin.Messaging.Ssh.MessagingScpClient.<UploadAsync>d__17.MoveNext() in D:\a\_work\1\s\src\Xamarin.Messaging.Ssh\MessagingScpClient.cs:line 70
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning : --- End of stack trace from previous location where exception was thrown ---
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
3>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Xamarin\iOS\Xamarin.Shared.targets(1641,3): warning :    at Xamarin.Messaging.Ssh.MessagingFileManager.<UploadFileAsync>d__25.MoveNext() in D:\a\_work\1\s\src\Xamarin.Messaging.Ssh\MessagingFileManager.cs:line 305
3>C:\Source\Repos\ClientMobileApp\Multipass.iOS\Main.cs(21,8,21,53): warning CS0618: 'UIApplication.Main(string[]?, string?, string?)' is obsolete: 'Use the overload with 'Type' instead of 'String' parameters for type safety.'
3>C:\Source\Repos\ClientMobileApp\Multipass.iOS\Services\ApplePayAuthorizerService.cs(65,36,65,55): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
3>  Multipass.iOS.KontronE -> C:\Source\Repos\ClientMobileApp\Multipass.iOS.KontronE\bin\iPhoneSimulator\Debug\Multipass.iOS.KontronE.exe
3>  /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/bin/mtouch @/Users/admin/Library/Caches/Xamarin/mtbs/builds/Multipass.iOS.KontronE/4f9fe3b4a459638c119440281481a8745fe7765b69b7afd2db772c1e546c45fe/obj/iPhoneSimulator/Debug/response-file.rsp 
========== Build: 3 succeeded, 0 failed, 1 up-to-date, 0 skipped ==========
========== Deploy: 0 succeeded, 0 failed, 0 skipped ==========
Launching 'Multipass.iOS.KontronE' on 'iPhone 13 iOS 15.2'...

</details>


<!--
### Screenshots
-->

<!-- If the issue is a visual issue, please include screenshots showing the problem if possible -->

<!--
### Reproduction Link
-->

<!-- Please upload or provide a link to a reproduction case -->

### Workaround

<!-- Did you find any workaround for this issue? This can unblock other people while waiting for this issue to be resolved -->

use Xamarin.Forms / Xamarin.Forms.Visual.Material Version=5.0.0.2291

--- a/Xamarin.Forms.Platform.iOS/Renderers/ImageRenderer.cs
+++ b/Xamarin.Forms.Platform.iOS/Renderers/ImageRenderer.cs
@@ -120,7 +120,7 @@ namespace Xamarin.Forms.Platform.iOS
 
 		void UpdateBackground()
 		{
-			var parent = Control.Superview;
+			var parent = Control?.Superview;
 
 			if (parent == null)
 				return;

